### PR TITLE
Add visible_only and strict_mode options to ClickTool

### PR DESCRIPTION
**Pull request by cancan101**
Partially addresses: https://github.com/hwchase17/langchain/issues/4066

### CODE REVIEW

File: langchain/tools/playwright/click.py [hunk 1]
```diff
@@ -26,6 +26,18 @@ class ClickTool(BaseBrowserTool):
     description: str = "Click on an element with the given CSS selector"
     args_schema: Type[BaseModel] = ClickToolInput
 
+    visible_only: bool = True
+    """Whether to consider only visible elements."""
+    strict_mode: bool = False
+    """Whether to employ Playwright's strict mode when clicking on elements."""
+    timeout: float = 1_000
```
Comment (vowelparrot) [inline langchain/tools/playwright/click.py:33]: What are the units of this? Could you add to the description?

Comment (dev2049) [reply langchain/tools/playwright/click.py:33]: personally don't feel need for prefix (all of this lives within playwright module after all), but don't feel strongly either way

Comment (cancan101) [reply langchain/tools/playwright/click.py]: Updated with units for timeout and I did end up adding the prefixes to signify those values are passed to the Playwright engine.

**Comment by dev2049** [comment]
thanks @cancan101!

**Pull request merged**